README.md

```diff
@@ -1,2 +1,39 @@
-# sass-bundle
-Delightful Sass Support for Symfony + AssetMapper
+# Sass For Symfony!
+
+This bundle make it easy to use Sass with Symfony's AssetMapper Component
+(no Node required!).
+
+- Automatically downloads the correct Sass binary
+- Adds a `sass:build` command to build and watch your Sass changes
+
+## Installation
+
+Install the bundle
+
+```shell
+composer require symfonycasts/sass-bundle
+```
+
+## Usage
+
+To use generated CSS, start by including the input file
+(`assets/styles/app.css` by default)
+
+```php
+{# templates/base.html.twig #}
+
+{% block stylesheets %}
+    <link rel="stylesheet" href="{{ asset('styles/app.scss') }}">
+{% endblock %}
+```
+
+Then you can create your SCSS file (`assets/styles/app.scss` by default)
+And start writing your style.
```

Comment:
This sentence will now be moved above (per my comment a few lines up). Replace it with something like:

```rst
That's right! You point directly to the `.scss` file. But don't worry, the final built `.css` file will be returned!
```